examples/functions/functions.md

```diff
@@ -30,11 +30,11 @@ fn full_name(first_name, last_name string) string {
 
 ## Variadic Functions
 
-Functions can also be variadic i.e. accept infinite number of arguments. They are **not** arrays and cannot be returned.
+Functions can also be variadic i.e. accept an infinite number of arguments. They are not arrays and cannot be returned.
```

Comment:
Dunno why `an` should be used here. I don't see the need to.

Reply:
Well, I think the `an` article is connected to the infinite number of arguments which is a group of arguments. Since it's not uncountable, I'd say the `an` should be required.